tests/nextflow.config

```diff
@@ -22,3 +21,5 @@ singularity.runOptions = "--containall"
 
 /* Remove gzipping on JSON output for testing/asserts on file contents */
 iridanext.output.path = "${params.outdir}/iridanext.output.json"
+
+process.errorStrategy = "terminate"
```

Comment:
Why do you have terminate as the default error strategy?

Reply:
If there was a larger reason I should have documented it. I beleive this was merely to have things fail faster as I was running tests locally. I will remove it!

Reply:
fixed in: [215a70b](https://github.com/phac-nml/mikrokondo/pull/96/commits/215a70b19612fe50ae48c5194cad0658665486b6)

Reply:
I really need to know how this keeps occuring... fixed it here: https://github.com/phac-nml/mikrokondo/pull/96/commits/034d9c88b91c3d9e311a93f66ef50534fce6ea99